hummingbird/ml/operator_converters/skl_iforest.py

```diff
@@ -0,0 +1,253 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converters for scikit-learn isolation forest
+"""
+
+from . import constants
+from hummingbird.ml.operator_converters._tree_implementations import TreeImpl, GEMMTreeImpl, TreeTraversalTreeImpl, \
+    PerfectTreeTraversalTreeImpl
+
+from ._tree_commons import get_parameters_for_sklearn_common, \
+    get_parameters_for_tree_trav_sklearn, get_tree_params_and_type, get_parameters_for_gemm_common
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+
+def _average_path_length(n_samples_leaf):
+    """
+    Taken from sklearn implementation of isolation forest:
+    https://github.com/scikit-learn/scikit-learn/blob/fd237278e/sklearn/ensemble/_iforest.py#L480
+    The average path length in a n_samples iTree, which is equal to
+    the average path length of an unsuccessful BST search since the
+    latter has the same structure as an isolation tree.
+    Parameters
```

Comment:
Can you refactor this doc a bit to follow the pdoc convention: Like using `Args:` and `Return:`

Reply:
will do